pyzeebe/channel/oauth_channel.py

```diff
@@ -0,0 +1,146 @@
+from functools import partial
+from typing import Optional
+
+import grpc
+from grpc.aio._typing import ChannelArgumentType  # type: ignore[import-untyped]
+
+from pyzeebe.credentials.oauth import Oauth2ClientCredentialsMetadataPlugin
+
+
+def create_oauth2_client_credentials_channel(
+    target: str,
+    client_id: str,
+    client_secret: str,
+    authorization_server: str,
+    scope: Optional[str] = None,
+    audience: Optional[str] = None,
+    channel_credentials: grpc.ChannelCredentials = grpc.ssl_channel_credentials(),
```

Comment:
```suggestion
    channel_credentials: Optional[grpc.ChannelCredentials] = None,
```

Reply:
To be honest I don't really know what's better.
My reasoning was that with the default value the user gets nudged in the right direction regarding which function to use.